src/commcare_cloud/ansible/roles/ecryptfs/tasks/wal.yml

```diff
@@ -0,0 +1,66 @@
+---
+- name: Check for already mounted encrypted wal drive
```

Comment:
instead of copying these can't we put them in a separate tasks file that we can include and use a var to tell it which drive to use:

```
include: encrypt_dir.yml
vars:
    path: "{{ ecrypted_wal_archive }}"
```
(not sure if that's the right was to pass vars)